reframework/autorun/UI_Scaler.lua

```diff
@@ -18,7 +18,8 @@ local settings = {
 local elementDatas = {
 	["snow.gui.GuiHud"] = {},
 	["snow.gui.GuiHud_QuestTarget"] = {},
-	["snow.gui.GuiHud_Timelimit"] = {},
+	["snow.gui.GuiHud_Timelimit"] = {anchor = "CenterTop"},
```

Comment:
As I mentioned unfortunately the time limit element is used for multiple things it seems so can't just move the anchor and call it done.
I think probably needs individual sub panels set up to work properly. Honestly might be worth just removing it from the list entirely. Idk. If you wanna figure out the panel settings you can submit that and it'd be pretty cool though.
Pretty annoying one to figure out tho considering it's only on screen for a few seconds. Though you could extend the timer in REF in the right place. 

I'm not really sure how to use github though. Is it possible to just not accept this part of the change? And what does this review thing do anyway.

Reply:
I think I got what you mean. Let me take a look at how this could be done.
By far, I found the timer is used for rampage countdown and return to village. Is there anything I'm missing?

Reply:
Who knows, those two are probably the only ones though.
Looking at the names of the subpanels might show something though.
Also I think the other change with the interval skip thing should be fine so can go ahead and keep that

Reply:
For example the rampage timer panel can be added like this.
![image](https://user-images.githubusercontent.com/11179201/152727971-d000516b-8211-4296-8538-d7c0176ba7ad.png)
This panel doesn't update it's own position every frame so if you use a posAdjust of >1 then it will drift away since its additive. Which could be solved with using absolute positioning and setting it to half 1080p on the X direction.
But its easier just to set the posAdjust multipliers to zero and it will scale in place where it starts.

Then you can add it to the default stuff by doing this
![image](https://user-images.githubusercontent.com/11179201/152728455-08d4874e-bb92-4d58-878e-bdd6b711ec03.png)

Then just do the same thing for the other panels.
Though I'm not sure what each of the rest of these are:
![image](https://user-images.githubusercontent.com/11179201/152728538-a4feda8d-ccdf-4256-b7cb-296a74d2ef82.png)

Reply:
also i've gone ahead and added the timer fixes with the panels myself so dont worry about it.